hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/SnapshotInfo.java

```diff
@@ -261,13 +264,15 @@ public SnapshotInfo.Builder toBuilder() {
         .setGlobalPreviousSnapshotId(globalPreviousSnapshotId)
         .setSnapshotPath(snapshotPath)
         .setCheckpointDir(checkpointDir)
+        .setDbTxSequenceNumber(dbTxSequenceNumber)
         .setDeepClean(deepClean)
         .setSstFiltered(sstFiltered)
         .setReferencedSize(referencedSize)
         .setReferencedReplicatedSize(referencedReplicatedSize)
         .setExclusiveSize(exclusiveSize)
         .setExclusiveReplicatedSize(exclusiveReplicatedSize)
-        .setDeepCleanedDeletedDir(deepCleanedDeletedDir);
+        .setDeepCleanedDeletedDir(deepCleanedDeletedDir)
+        .setLastTransactionInfo(lastTransactionInfo);
```

Comment:
What about the existing snapshots when the system is upgraded ? they would not have this field set.

Reply:
The lastTransactionInfo would be having a null value